src/utils/api.js

```diff
@@ -26,13 +26,22 @@ const handleResponse = (request) =>
     },
   )
 
+export const LOCALE = {
+  EN: 'en',
+  DE: 'de',
+  EL: 'el',
+  ES: 'es',
+  FR: 'fr',
+  HE: 'he',
+  IT: 'it',
+  NL: 'nl',
+  PL: 'pl',
+}
+
 export const getClusters = (
-  swlng,
-  nelng,
-  swlat,
-  nelat,
-  zoom = null,
-  muni = null,
+  { swlng, nelng, swlat, nelat },
+  zoom = 0,
+  muni = false,
```

Comment:
We should change this to `includeMunicipal` to be consistent with the above code.